crates/shared/src/lib.rs

```diff
@@ -46,3 +48,17 @@ pub mod zeroex_api;
 pub fn clone_anyhow_error(err: &anyhow::Error) -> anyhow::Error {
     anyhow::anyhow!("{:#}", err)
 }
+
+pub fn random_account() -> Account {
```

Comment:
nit: it's not really a random account (it will be the same everytime the code is run). Instead of creating this logic and unnecessary loop, can we not simply hardcode return some address we deem feasible (feel free to make a hex pun)